docs/ref/forms/fields.txt

```diff
@@ -494,6 +494,11 @@ For each field, we describe the default widget used if you don't specify
         time the field's form is initialized, in addition to during rendering.
         Defaults to an empty list.
 
+    .. note::
```

Comment:
To improve readability, use `.. admonition:: Descriptive title` rather than `.. note::` as described in [Guidelines for reStructuredText files](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files).